projects/novo-elements/src/elements/dropdown/Dropdown.ts

```diff
@@ -194,7 +208,7 @@ export class NovoDropdownElement implements OnInit, OnDestroy {
       this.filterTermTimeout = setTimeout(() => {
         this.filterTerm = '';
       }, 2000);
-      const char = String.fromCharCode(event.keyCode);
+      const char = event.key;
```

Comment:
I was hoping for Angular Cdk to show us how to move away from `keyCode` https://github.com/angular/components/issues/7374
But apparently they are not ready yet.